aws_marketplace/curating_aws_marketplace_listing_and_sample_notebook/Algorithm/curating_good_algorithm_listing.md

```diff
@@ -0,0 +1,46 @@
+| #   | Section          | Description                                                                                                             | Mandatory/Highly Recommended | Sample Example                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+|-----|------------------|-------------------------------------------------------------------------------------------------------------------------|------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| PO1 | Product overview | Describe algorithm category e.g., Tree, Neural Net etc. Ensemble                                                        | Mandatory                    | An AutoML algorithm that trains a multi-layer stack ensemble model to predict on regression/classification datasets directly from CSV data.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
```

Comment:
It is suggested to use "for example" rather than "e.g.". However, if e.g. is to be used, it should be in parentheses:

```suggestion
| PO1 | Product overview | Describe algorithm category (e.g., Tree, Neural Net, Ensemble)                                                        | Mandatory                    | An AutoML algorithm that trains a multi-layer stack ensemble model to predict on regression/classification datasets directly from CSV data.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
```